android/sdl_android/src/androidTest/java/com/smartdevicelink/test/Validator.java

```diff
@@ -3929,4 +3929,14 @@ public static boolean validateStabilityControlStatus(VehicleDataResult status1,
         }
         return status1.getDataType().equals(status2.getDataType()) && status1.getResultCode().equals(status2.getResultCode());
     }
+
+    public static boolean validateClimateData(ClimateData climateData1, ClimateData climateData2) {
```

Comment:
@kostyaBoss I would rewrite the `validateClimateData` function like this. 

    `public static boolean validateClimateData(ClimateData climateData1, ClimateData climateData2) {
        if (climateData1 == null) {
            return (climateData2 == null);
        }
        if (climateData2 == null) {
            return (climateData1 == null);
        }

        if (!validateTemperature(climateData1.getExternalTemperature(), climateData2.getExternalTemperature())) {
            return false;
        }

        if (!validateTemperature(climateData1.getCabinTemperature(), climateData2.getCabinTemperature())) {
            return false;
        }

        return climateData1.getAtmosphericPressure().floatValue() == climateData2.getAtmosphericPressure().floatValue();
    }`